test/local/email_record_tests.js

```diff
@@ -216,9 +216,9 @@ test(
     er.addHit()
     er.addHit()
     t.equal(er.shouldBlock(), false, 'account is not blocked')
-    t.equal(er.update('accountCreate'), 0, 'email action above the email limit')
+    t.equal(er.update('accountCreate'), 1, 'email action above the email limit')
     t.equal(er.shouldBlock(), true, 'account is now blocked')
-    t.equal(er.update('accountCreate'), 0, 'email action in a blocked account')
+    t.equal(er.update('accountCreate'), 1, 'email action in a blocked account')
```

Comment:
We can see here that several tests have hacked around this problem in the past.  The comment says we're asserting that an action was blocked, but `er.update` is returning zero which is supposed to indicate success.